docs/guide/test.md

```diff
@@ -0,0 +1,51 @@
+<script setup>
```

Comment:
can export in `src/client/theme-default/index.ts`
and register in `src/client/app/index.ts` as a global component
so that we don't need import in every page